### ISSUE TITLE
[EXPERIMENT] FCN - Full-Resolution training 

### ISSUE THREAD

**Issue by refor53** [boostcampaitech7/level2-cv-semanticsegmentation-cv-01-lv3#16]
## 📌 가설 (Hypothesis)

- 가설: image 사이즈를 720으로 키워 성능 향상을 보인 팀원의 실험을 보고, 최대 이미지(원본, 2048)로 실험을 진행해보고자 함. 

---

## ✅ 어떻게 실험을 진행할 것 인가요?
 1. 이미지 사이즈를 2048로 변경
 2. 이에 맞춰 배치사이즈를 허용가능한 만큼 변경(max_batch = 1)
 3. batch_size에 대한 보상으로 Gradient accumulation 진행

### 전처리 기법 (optional) 

- 

---

### 정량적 결과

- 

### 정성적 결과

- 

### 결론

- 

---
